cirq/google/sim/mem_manager_test.py

```diff
@@ -79,6 +80,8 @@ def test_fills_gaps():
     mem_manager.SharedMemManager.free_array(new_handle)
 
 
+@pytest.mark.skipif(sys.platform.startswith('win'),
+                    reason="skipping linux-only tests")
```

Comment:
"skipping linux-only tests" makes it sound like it is skipping linux tests. 
I'd make the reason something more about why this test
"skipping test on windows due to lack of multiprocessor support"

Reply:
Done.